### PR TITLE
Revised class library getting started topics for .NET Core 2.0

### PR DESCRIPTION
**Pull request by rpetrusha**
# Revised class library getting started topics for .NET Core 2.0

Also adds a VB class library tutorial, and uses tabs for the test tutorial. (The consuming a class library tutorial is will come shortly.)

I'll do TOC modifications and redirects in a separate PR.

## Suggested Reviewers

@mairaw  @BillWagner

### CODE REVIEW

**Comment by mairaw** [comment]
@BillWagner @rpetrusha I have rebased this branch to resolve the merge conflicts

File: docs/core/tutorials/index.md [hunk 1]
```diff
@@ -40,7 +41,7 @@ The following tutorials are available for learning about .NET Core.
 
 - [Getting started with .NET Core on Windows/Linux/macOS using the .NET Core CLI tools](using-with-xplat-cli.md)
 - [Organizing and testing projects with the .NET Core CLI tools](testing-with-cli.md)
-- [Getting started with F#](../../fsharp/tutorials/getting-started/getting-started-command-line.md)
+- [Getting started with F#](../../fsharp/tutorials/getting-started/get-started-command-line.md)
```
Comment (BillWagner) [inline docs/core/tutorials/index.md]: Should the TOC node also change to 'Get started with F#'?

File: docs/core/tutorials/testing-library-with-visual-studio.md [hunk 2]
```diff
@@ -44,9 +46,39 @@ To create the unit test project, do the following:
...
+1. In **Solution Explorer**, open the context menu for the **ClassLibraryProjects** solution node and select **Add** > **New Project**.
+
+1. In the **Add New Project** dialog, select the **Visual Basic** node. Then select the **.NET Core** node followed by the **Unit Test Project (.NET Core)** project template. In the **Name** text box, enter "StringLibraryTest" as the name of the project. Select **OK** to create the unit test project.
+
+   ![Add New Project dialog](./media/testing-library-with-visual-studio/vb-testproject.png)
+
+   > [!NOTE]  
+   > In addition to a Unit Test project, you can also use Visual Studio to create an xUnit test project for .NET Core.
+
+1. Visual Studio creates the project and opens the *UnitTest1.cs* file in the code window.
```
Comment (BillWagner) [inline docs/core/tutorials/testing-library-with-visual-studio.md]: In the VB section, shouldn't this be *TestLibrary.vb*?

**Review by BillWagner — APPROVED** [review]
Overall, this looks great @rpetrusha 
I found one error where a VB file is referred to as a CS file. After that, it's time to :shipit:

File: docs/core/tutorials/index.md (hunk 1 above)
Comment (rpetrusha) [inline docs/core/tutorials/index.md]: @BillWagner, yes, I'll do this in a separate PR.

**Comment by rpetrusha** [comment]
@mairaw, thanks for rebasing the branch for me!

**Pull request merged**